packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -1,9 +1,60 @@
-import { Type, restBaseUrl, validators } from '@openmrs/esm-framework';
+import { Type, validators } from '@openmrs/esm-framework';
 import vitalsConfigSchema, { type VitalsConfigObject } from './current-visit/visit-details/vitals-config-schema';
 import biometricsConfigSchema, {
   type BiometricsConfigObject,
 } from './current-visit/visit-details/biometrics-config-schema';
 
+export const defaultTablesConfig: TablesConfig = {
```

Comment:
Definition of default columns to show in the queue table of the service queues app main page.